packages/framework/src/Model/Customer/User/CustomerUserRepository.php

```diff
@@ -196,4 +196,23 @@ public function getOneByUuid(string $uuid): CustomerUser
 
         return $customerUser;
     }
+
+    /**
+     * @param int $salesRepresentativeId
+     * @return array
```

Comment:
this returns an array of emails, right? So I would use here `@return string[]` and consider naming the method accordingly

Reply:
done, opted for findEmailsOfCustomerUsersUsingSalesRepresentative, hope it is ok

Reply:
name is ok, please, fix the return annotation to `string[]` yet

Reply:
fixed